samples/sensor/ti_hdc/nucleo_l496zg.overlay

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2019 Centaur Analytics, Inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&i2c1 {
```

Comment:
Is that arduino_spi ?
If it is, maybe this can be moved to something more generic

Reply:
renamed to **arduino_i2c**, is that ok?

Reply:
@erwango ping